### DISCUSSION TITLE
Meadow F7 Support

### DISCUSSION THREAD

**Discussion by MarkAlanJones** [robhagemans/hoard-of-bitfonts#10] [General]
[Meadow Font Test](https://github.com/MarkAlanJones/MeadowFontTest)  (Yaff support is near the bottom)

I have a C# Yaff font reader, that can be used to display Yaff Fonts on the Meadow F7 platform, using the platform's Micrographics library DrawText Functions.  i.e. it supports the Meadow's IFont for fixed width's fonts 12px or less

I hope to liberate some proportional fonts next

Thanks for collecting the fonts, and the Yaff format is quite useful
Mark Jones

**Comment by robhagemans** [comment]
Hey, that's cool, thanks for letting me know!
